lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -165,7 +165,9 @@ describe('CategoryRenderer', () => {
 
     assert.ok(pwaCategory.manualDescription);
     const description = categoryDOM
-      .querySelector('.lh-clump--manual .lh-audit-group__description').textContent;
+      .querySelector('.lh-clump--manual .lh-audit-group__description').textContent
+      // remove " — "
+      .substring(3);
```

Comment:
nit: this feels a tad brittle, maybe a regex match or something at least?

Reply:
done